### ISSUE TITLE
Unable to publish the presence status

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Salavan/sipml5#57]
```
I am trying to change the code for publishing the status of the logged in user.

I have attached the file I have changed. When I call the method for sendStatus 
it does not send the sip message.

Is there code included for publishing the status. If yes, Please let me know as 
I am trying to learn Sipml5.


```

Original issue reported on code.google.com by `amol8...@gmail.com` on 22 Nov 2012 at 9:32

Attachments:
- [tsip_api_publish.js](https://storage.googleapis.com/google-code-attachments/sipml5/issue-57/comment-0/tsip_api_publish.js)

**Comment by GoogleCodeExporter** [comment]
```
Achieved the publish functionality, cant commit the code having it with me. 
Thanks.
```

Original comment by `amol8...@gmail.com` on 29 Nov 2012 at 1:14

**Comment by GoogleCodeExporter** [comment]
```
Fixed in API v1.1.0 (SVN r148)
To download API: http://code.google.com/p/sipml5/wiki/Downloads
Sample code: http://sipml5.org/docgen/index.html#publishPresence
```

Original comment by `boss...@yahoo.fr` on 20 Dec 2012 at 7:15
- Changed state: **Fixed**
